algosdk/v2client/algod.py

```diff
@@ -1,14 +1,10 @@
-from urllib.request import Request, urlopen
```

Comment:
I re-org'ed the imports when I touched this file to add functionality to `app_info`. That functionality was reverted but I'm asking to keep the import cleanup.